### PR TITLE
Add injective-py

### PR DESCRIPTION
**Pull request by MementoRC**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by MementoRC** [comment]
@conda-forge/staged-recipes I cannot understand why this package fails to build on windows. It fails on `eip712`, for which I have a standalone recipe in draft that builds without issue: https://github.com/conda-forge/staged-recipes/pull/23788.
It looks like it is a recipe environment issue. There are warnings about not being able to unlink temp directories. There is a `conda.tmp` permission denied (that does not happen with the standalone recipe), then the `dist-info` build fails with apparently not being able to create the __bkp__ directory

Note also, that at some point, a windows build complained about `htbuilder`, when nowhere do I reference it, later I saw a github merge referencing `htbuilder` - It seems that there are cross-pollution of built packages on windows (that may be completely irrelevant, but I was very surprised to see a reference to `htbuilder` and thought to mention it)

**Comment by carterbox** [comment]
Awaiting #23787

**Comment by MementoRC** [comment]
@carterbox It is awaiting `coincurve` upgrade and I contacted the author who is having some issues with windows build using the static library. Can I update `bip32` to coincurve 19.0.0 and unblock both `bip32` and `injective-py`?

**Comment by MementoRC** [comment]
@carterbox It looks ready for review to me. Thx

File: recipes/injective-py/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+version:
+  - 0.9.12
```
Comment (xhochy) [inline recipes/injective-py/conda_build_config.yaml:2]: Please build different versions as separate branches on the feedstock and not via this way.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/injective-py/conda_build_config.yaml (hunk 1 above)
Comment (MementoRC) [reply recipes/injective-py/conda_build_config.yaml:2]: Sure.

**Review by xhochy — APPROVED** [review]

**Pull request merged**